great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py

```diff
@@ -205,6 +205,19 @@ def _generate_batch_spec_parameters_from_batch_definition(
             **self.data_assets[data_asset_name],
         }
 
+    def _validate_batch_request(self, batch_request: BatchRequest):
```

Comment:
Not sure if this is the right place...  We are deep inside the SQL connector...  I feel that `BatchRequest` "basics" have to be validated up front.

Reply:
this has been changed in the update. closing